ocpmodels/common/relaxation/optimizers/lbfgs_torch.py

```diff
@@ -95,7 +95,7 @@ def run(self, fmax, steps):
         if self.traj_dir:
             self.traj_dir.mkdir(exist_ok=True, parents=True)
             trajectories = [
-                ase.io.Trajectory(self.traj_dir / f"{name}.traj", mode="a")
+                ase.io.Trajectory(self.traj_dir / f"{name}.traj_tmp", mode="w")
```

Comment:
Just confirming: with `mode="w"`, this will overwrite any previously existing .traj_tmp file, but when we do `traj.write(atm)` it will append the current iteration's atoms to the in progress trajectory, right?

Reply:
I had the same question, tested it on my end and it's appending as expected.